src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
-package core.basesyntax;
+package main.java.core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        new Truck();
+        new Buldozer();
+        new Excavator();
 
+        Machine.startAllMachines();
+        Machine.stopAllMachines();
```

Comment:
let's remove these static methods from`Machine`.  Instead of it use the capabilities of polymorphism. Create an array of `Machine` and pass implementations of it in this array in `main()` method.  After that you can go through this array, using for loop and call methods `doWork()` and `doStop()` in it.